packages/backend/src/api/apiV1.ts

```diff
@@ -35,7 +37,7 @@ apiV1Router.post('/register', unauthorisedInDemo, async (req, res, next) => {
             isMarketingOptedIn: !!req.body.isMarketingOptedIn,
             isTrackingAnonymized: !!req.body.isTrackingAnonymized,
         });
-        const sessionUser = await UserModel.findSessionUserByUUID(
```

Comment:
@ZeRego are we happy to call the model directly in the route handler? Would we prefer this to move into the service?

Reply:
I'm split on this one. I would like for routes to only interact with services but this is going to add lots of boilerplate where the service method is just calling the model method.

Reply:
Eventually we need to stop calling models directly since we need to track who is accessing what resource. The service is where we check permissions and record audit logs.